lsp/angularls.lua

```diff
@@ -15,7 +15,7 @@
 -- Angular requires a node_modules directory to probe for @angular/language-service and typescript
 -- in order to use your projects configured versions.
 local root_dir = vim.fn.getcwd()
-local node_modules_dir = vim.fs.find('node_modules', { path = root_dir, upward = true })[1]
+local node_modules_dir = vim.fs.find('node_modules', { path = root_dir })[1]
```

Comment:
I think I mentioned when this config was first created, that getcwd() in the module is only going to accidentally work if the user starts Nvim in the right directory. That's a rather flimsy assumption. It's not surprising that this config has bugs.